dcrjson/walletsvrresults.go

```diff
@@ -163,6 +165,44 @@ type ListScriptsResult struct {
 	Scripts []ScriptInfo `json:"scripts"`
 }
 
+// ListTicketsResult models the data from the listtickets command.
+type ListTicketsResult struct {
+	Ticket  *ListTicketsTransactionSummary `json:"ticket"`
+	Spender *ListTicketsTransactionSummary `json:"spender"`
+	Status  string                         `json:"status"`
+}
+
+// ListTicketsTransactionSummary defines the type used in the listtickets JSON-RPC
+// result for the Ticket and Spender command fields.
+type ListTicketsTransactionSummary struct {
+	Hash        string                                `json:"hash"`
+	Transaction string                                `json:"transaction"`
+	MyInputs    []ListTicketsTransactionSummaryInput  `json:"my_inputs"`
+	MyOutputs   []ListTicketsTransactionSummaryOutput `json:"my_outputs"`
```

Comment:
I think the "`My`" prefix is confusing.  These would be clear as just `Inputs` and `Outputs`